### DISCUSSION TITLE
config page无法显示配置文件内容

### DISCUSSION THREAD

**Discussion by cuitzeno** [whyour/qinglong#1934] [Q&A]
今天更新最新版青龙：2.15.14，配置文件页面（http://xxx.xxx.xxx.xxx/config）无法加载文件内容是怎么回事？

系统信息
版本v2.15.14
更新时间2023-05-13 01:11:03
更新IDa596241

<img width="1428" alt="image" src="https://github.com/whyour/qinglong/assets/34862910/cd2d96bb-f76b-4731-b8b9-e01968e944ce">

**Comment by Cayde23** [comment]
换了服务器，今天重新安装青龙面板时遇到了一样的问题![example](http://101.42.224.58:15576/%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20230513174431.png)

**Comment by cuitzeno** [comment]
发了新版解决了这个问题，去更新吧。。。
